components/css/newsletter-text-only/_contentPage.scss

```diff
@@ -2,172 +2,194 @@ $white:#ffffff;
-	line-height: 1.3em;
-	text-align: left;
-	padding: 1em 0 .4em 0;
-	}
 	
-	&__subtitle {
-	color: $light-gray;
-	font-size: 1em;
-	line-height: 1.5em;
-	text-align: left;
-	padding: 0 0 .8em 0;
-	}
+		&__title {
+			color: $campusGold;
+			font-size: 1.4em;
+			line-height: 1.3em;
+			padding: 1em 0 0 0;
+				&-light {
+					color: #000000;
+				}
+		}
 	
-	&__teaser {
-	color: $white;
-	font-size: .9em;
-	line-height: 1.5em;
-	text-align: left;
-	padding: 0 0 .5em 0;
-	}
-}	
-
-
-.list {
-	list-style-type: none;
-	width: 330px;
-	padding: 1.5em 1.5em 0 .5em;
+		&__subtitle {
+			color: $light-gray;
+			font-size: 1em;
+			line-height: 1.5em;
+			padding-top: .5em;
+			padding-bottom: .8em;
+		}
 	
-
-	li {
-	color: $gold;
-	line-height: 1.5em;
-	display: flex;
-	align-items: center;
-	justify-content: center;
-	}
-
-	li span:first-child {
-	flex-grow: 0;
-	flex-shrink: 0;
-	flex-basis: 3em;
-	}
-
-	li span:last-child {
-	flex-grow: 1;
-	font-size: .9em;
-	line-height: 1.4em;
-	}
-
-	.fa-2x {
-	width: 3em;
-	color: $black;
-	margin: 0 0 0 -1em;
-	}
-
-	.fa-newspaper-o {
-	color: $light-gray;
-	}
+		&__teaser {
+			color: $white;
+			font-size: .9em;
+			line-height: 1.5em;
+			padding-bottom: .5em;
+		}
 	
-	&--light {
-		background-color:$white;	
+		&--light {
+			background-color:$white;	
+		}
+	
+		&__link {
+			font-size: .9em;
+			line-height: 1.4em;
+				&-other {   
+					display:flex;
+					align-items:center;
+					margin-top: 20px;
+        
+				&:link,&:visited,&:hover,&:active{
+					color:$campusGold;
+					text-decoration:none;
+				}
+					
+				&::before{
+					content:"\f1ea";
+					font-family:FontAwesome;
+					display:flex;
+					flex-grow:0;
+					flex-shrink:0;
+					flex-basis:auto;
+					justify-content:center;
+					align-items:center;
+					float:left;
+					margin-right:10px;
+					padding:0;
+					height:2em;
+					width:2em;
+					border-radius:50%;
+					background-color:$black;
+					color: $light-gray;
+					font-size:2em;
+					line-height:1em;
+					/* other styles etc. */
+				}
+        	}
+    	}
 	}
-}
+
 
 .button {
-	margin: 1em 0 0 0 !important;
+	margin-top: 1em !important;
 }
 
-.cta-archive {
+.cta {
 	width: 100%;
 	padding: 1em;
 	background: repeating-linear-gradient(60deg,#707070,#707070 6px,#777777 6px,#777777 8px);
 	
-	&__title {
-	color: $white;
-	font-size: 1.5em;
-	line-height: 1.5em;
-	text-align: center;
-	padding: 0;
-	}
-	
-	&__description {
-	color: #ffffff;
-	font-size: 1em;
-	line-height: 1em;
-	text-align: center;
-	padding-top: 0;
-	padding-bottom: 1.5em;
-	}
-	
-	.btn-center {
-	margin: auto !important;
-	display: block;
-	width: 250px;
-	}
-}
\ No newline at end of file
+		&__title {
+			color: $white;
+			font-size: 1.5em;
+			line-height: 1.5em;
+			text-align: center;
+			padding: 0;
+		}
+
+		&__description {
+			color: $white;
+			font-size: 1em;
+			line-height: 1em;
+			text-align: center;
+			padding-top: 0;
+			padding-bottom: 1.5em;
+		}
+
+		.btn-center {
```

Comment:
@jacobvenable is this the best way to center this button? We weren't sure which route to take.

Reply:
Instead of adding a new class, you could add the Bootstrap helper class, `center-block` on the element: https://getbootstrap.com/docs/3.3/css/#helper-classes-center